src/services/TensCheckService.ts

```diff
@@ -0,0 +1,24 @@
+interface PageDataResponse extends Object {
+  IsNormalAuthentication: boolean;
+  PostAuthAction: string;
+}
+
+async function RunTensCheck(): Promise<boolean> {
+  const dataPage: Promise<PageDataResponse> = PCore.getDataPageUtils().getPageDataAsync(
+    'D_PostCitizenAuthAction',
+    'root'
+  ) as Promise<PageDataResponse>;
+  const response = await dataPage;
+  console.log('RESPONSE::::', response);
+  if (response?.IsNormalAuthentication === false) {
+    const currentPage = window.location.href;
+    window.location.replace(
+      `https://www.tax.service.gov.uk/protect-tax-info?redirectUrl=${currentPage}`
```

Comment:
I think it's available in stg1. URL we need to configure like below  - 
https://www.**staging**.tax.service.gov.uk/protect-tax-info?redirectUrl=**https://account-np.hmrc.gov.uk/child-benefit/make_a_claim/test/**

note - redirectUrl param can be dynamic 

Refer below component -
components\BaseComponents\MainWrapper\index.tsx

Reply:
Where did you find this?
No one has heard of this url

Reply:
The staging and QA, to my understanding, do not have the TENS check.